src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -726,11 +703,13 @@ private void removeTerritoriesWhereTransportsAreExposed(final Map<Territory, Pro
               defenders.addAll(moveMap.get(unloadTerritory).getMaxUnits());
             }
             final ProBattleResult result =
-                battleUtils.calculateBattleResults(player, unloadTerritory, enemyAttackOptions.getMax(unloadTerritory)
-                    .getMaxUnits(), new ArrayList<Unit>(defenders), new HashSet<Unit>(), false);
+                ProBattleUtils.calculateBattleResults(player, unloadTerritory,
```

Comment:
boolean value parameter code smell. By definition SRP is violated if there are many parameters and one of them is a boolean. Can be fixed by removed the boolean parameter and breaking up the public API to have two methods instead of one.